docs-ref-services/latest/mysql.md

```diff
@@ -31,14 +31,15 @@ This example connects to a MySQL database and performs a simple query to retriev
 
 ```JavaScript
 const mysql = require('mysql2');
+const fs = require('fs');
 
 const connection = mysql.createConnection({
   host: 'mysqldemo.mysql.database.azure.com',
   user: 'myadmin@mysqldemo',
   password: 'your_password',
   database: 'my_db',
   port: 3306,
-  ssl: true
+  ssl: {ca: fs.readFileSync("your_path_to_ca_cert_file_BaltimoreCyberTrustRoot.crt.pem")}
```

Comment:
```suggestion
  ssl: {ca: fs.readFileSync("your_path_to_ca_cert_file.crt.pem")}
```

Reply:
`BaltimoreCyberTrustRoot` is confusing me a bit, and I'd prefer it get removed. However, the intent of this string is obvious, so I won't block on it.